src/test/scala/com/databricks/spark/xml/XmlSuite.scala

```diff
@@ -17,21 +17,21 @@ package com.databricks.spark.xml
 
 import java.io.File
 import java.nio.charset.UnsupportedCharsetException
+import java.nio.file.Files
 import java.sql.{Date, Timestamp}
 
 import scala.io.Source
 
-import org.scalatest.{BeforeAndAfterAll, FunSuite}
 import org.apache.hadoop.conf.Configuration
-import org.apache.hadoop.io.LongWritable
-import org.apache.hadoop.io.Text
+import org.apache.hadoop.io.{LongWritable, Text}
 import org.apache.hadoop.io.compress.GzipCodec
+import org.scalatest.{BeforeAndAfterAll, FunSuite}
 
-import org.apache.spark.{SparkException, SparkContext}
-import org.apache.spark.sql.{SaveMode, Row, SQLContext}
-import org.apache.spark.sql.types._
 import com.databricks.spark.xml.XmlOptions._
 import com.databricks.spark.xml.util.ParseModes
+import org.apache.spark.sql.types._
+import org.apache.spark.sql.{Row, SQLContext, SaveMode}
+import org.apache.spark.{SparkConf, SparkContext, SparkException}
```

Comment:
Let's keep the import order according to https://github.com/databricks/scala-style-guide#imports

Reply:
Fixed